framework/merchant_api/domain/entities/dora_product.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from uuid import UUID
+
+
+@dataclass
+class DoraProduct:
```

Comment:
Should MAPI have a concept of 'Dora' product?

Reply:
Or rather refined product or something

Reply:
yeah so i had this as "Product"  at first, but then realised it's kind of like Databuild Gateway having a "DatabuildOrder" entity. It's an "Order" that has come from Databuild.

Reply:
Ye guess that's true. Also now that i think of it this is more explanatory for the shape of the data rather than 'refined'product.